pyproject.toml

```
@@ -0,0 +1,25 @@
+[project]
+name = "akatsuki-utils-py"
+authors = [
+    {name = "Akatsuki Team", email = "support@akatsuki.gg"},
+]
+description = "Shared library for Akatsuki python apps"
+license = {text = "MIT"}
+readme = "README.md"
+requires-python = ">=3.7"
```

3.7 is past eol nowadays -- most of our apps are already running 3.10/3.11
```suggestion
requires-python = ">=3.9"
```

---

https://devguide.python.org/versions/